tensorboard/webapp/app_routing/store/app_routing_selectors.ts

```diff
@@ -100,8 +103,12 @@ export const getExperimentIdToAliasMap = createSelector(
     }
 
     const compareParams = routeParams as CompareRouteParams;
-    const map = getCompareExperimentIdAliasSpec(compareParams);
-    return Object.fromEntries(map.entries());
+    // We still need to use this internally until while switching to the new
+    // method.
+    // tslint:disable:deprecation
+    const userDefinedAliasMap = getCompareExperimentIdAliasSpec(compareParams);
+    // tslint:enable:deprecation
```

Comment:
OSS TB does not use tslint so this is vestigial; we do not use something that is not enforceable.

Reply:
Removed.